### PR TITLE
API Boilerplate

### PR DESCRIPTION
**Pull request by fishnix**

### CODE REVIEW

File: api/handlers_test.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/YaleSpinup/ec2-api/common"
+)
+
+func TestPingHandler(t *testing.T) {
+	req, err := http.NewRequest("GET", "/v1/ec2/ping", nil)
```
Comment (tenyo) [inline api/handlers_test.go:28]: 👀

File: api/handlers_test.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+package api
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/YaleSpinup/ec2-api/common"
```
Comment (tenyo) [inline api/handlers_test.go:24]: 👀

File: api/handlers_test.go [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+	expected := `pong`
+	if rr.Body.String() != expected {
+		t.Errorf("handler returned unexpected body: got %v want %v",
+			rr.Body.String(), expected)
+	}
+}
+
+func TestVersionHandler(t *testing.T) {
+	req, err := http.NewRequest("GET", "/v1/ec2/version", nil)
```
Comment (tenyo) [inline api/handlers_test.go:51]: 👀 I think I noted these in the api-tmpl already

Comment (fishnix) [reply api/handlers_test.go:51]: ah thanks yeah, probably needs to get fixed in the template.  i swear i wasnt re-writing the ec2-api in go 😱 😆

Comment (tenyo) [reply api/handlers_test.go:51]: yeah, sure 😆

**Pull request merged**